src/core/transactionmanager.js

```diff
@@ -539,6 +540,10 @@ TransactionManager.prototype._prepareRequest = function(tx) {
     submitRequest.txJson(tx.tx_json);
   }
 
+  if (!_.isEmpty(tx.signers)) {
+    submitRequest.message.Signers = tx.signers;
```

Comment:
Does that mean you eventually successfully got a regular `submit` to work with a multisigned blob?  If that doesn't work work I'll need to debug it.  But @sublimator is right, if you submit a blob the `Signers` array must be sorted.